### ISSUE TITLE
proxy not found for local queries: Missing import in urifunctions.py

### ISSUE THREAD

**Issue by astroclark** [ligovirgo/dqsegdb#13]
Local (i.e., non-LDG) segdb queries fail, despite a valid grid proxy:

```bash
[jclark308@PHYS44237] $ ligolw_segment_query_dqsegdb --ping -t https://segments.ligo.org
/home/jclark308/.local/lib/python2.7/site-packages/dqsegdb/urifunctions.py:39: UserWarning: Warning:  No proxy found or other error encountered during check for authentication credentials, connections to https will not work.
  warnings.warn("Warning:  No proxy found or other error encountered during check for authentication credentials, connections to https will not work.")
---
# Manual hack because the warnings.warn() line fails, too
# print url
# print e.reason
https://segments.ligo.org/dq/L1
[Errno 336265218] _ssl.c:351: error:140B0002:SSL routines:SSL_CTX_use_PrivateKey_file:system lib
---
Traceback (most recent call last):
  File "/home/jclark308/.local/bin/ligolw_segment_query_dqsegdb", line 721, in <module>
    json_out=urifunctions.getDataUrllib2(url_result)
  File "/home/jclark308/.local/lib/python2.7/site-packages/dqsegdb/urifunctions.py", line 117, in getDataUrllib2
    warnings.warn("Issue accesing url: %s; Reason: %s" % (url,str(e.reason)))
AttributeError: 'bool' object has no attribute 'warn'
```

The following reveals a missing import in urifunctions.py:
```python
python -c "from dqsegdb.urifunctions import findCredential; print(findCredential())"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/jclark308/.local/lib/python2.7/site-packages/dqsegdb/urifunctions.py", line 410, in findCredential
    if validateProxy(path):
  File "/home/jclark308/.local/lib/python2.7/site-packages/dqsegdb/urifunctions.py", line 455, in validateProxy
    if re.search(r'.+CN=proxy$', subject):
NameError: global name 're' is not defined
```

Solution is to add the re module to urifunctions.py to get
```bash
[jclark308@PHYS44237] $ python -c "from dqsegdb.urifunctions import findCredential; print(findCredential())"
('/tmp/x509up_u1000', '/tmp/x509up_u1000')
[jclark308@PHYS44237] $ ligolw_segment_query_dqsegdb --ping -t https://segments.ligo.org 
Server version is 2.1.16
```

**Comment by duncanmmacleod** [comment]
I think the missing import was resolved by 3411a8eeb3c50b81a09c3febc62bf2f07ea83c8a, @rpfisher please confirm and close.

**Comment by rpfisher** [comment]
Yes, this commit fixed that issue:  3411a8eeb3c50b81a09c3febc62bf2f07ea83c8a
